cypress/e2e/ApplicationNewPage.e2e.js

```diff
@@ -22,312 +22,312 @@ import { usingFixtures } from '../support/utils'
 describe('ApplicationNewPage', () => {
   beforeEach(() => {
     if (usingFixtures()) {
-      //   cy.intercept('GET', 'api/v1/short-form/**', { fixture: 'shortFormGet.json' }).as(
```

Comment:
Everything in this test got commented out mistakenly previously - just restoring it